docs/install/debug.rst

```diff
@@ -4,15 +4,50 @@
 Debugging
 =========
 
-.. Note:: 2019-08-16: This needs to be updated.
+Debugging webapp
+================
```

Comment:
This looks like two `<h1>` sections with no content between - do we need to adjust heading levels?

Reply:
The first one has over and under and is the header for the entire page, but it's hard to see because the previous line is hidden.